static/customWidgets/GeometryWidget/geometrywidget.js

```diff
@@ -269,6 +251,105 @@
 
     }
 
+    geometryWidget.prototype.setDrawingStyle = function() {
+      var imagePoint = new ol.style.Circle({
+        radius: 5,
+        stroke: new ol.style.Stroke({
+          color: 'rgba(0, 24, 110, 0.9)',
+          width: 2
+        }),
+        fill: new ol.style.Fill({
+          color: 'rgba(0, 48, 220, 0.6)',
+        })
+      });
+      var imageVertex = new ol.style.Circle({
+        radius: 3,
+        stroke: new ol.style.Stroke({
+          color: 'rgba(255, 24, 0, 0.8)',
+          width: 2
+        }),
+        fill: new ol.style.Fill({
+          color: 'rgba(255, 24, 0, 0.2)',
+        })
+      });
+      return [
+        new ol.style.Style({
+            image: imageVertex,
+            geometry: function(feature) {
+              var geomType = feature.getGeometry().getType();
+              var geomCoords = feature.getGeometry().getCoordinates();
```

Comment:
Causes a TypeError: feature.getGeometry(...).getCoordinates is not a function ERROR if geometry type does not have a getCoordinates() fonction! moving that inside the if structure should do the trick